### ISSUE TITLE
webtide() should be quieter in checking for data files

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1165]
Since the webtide application may put its data in any of several spots, `oce::webtide` has to run through a list. At the moment, it spits a warning for each file that it fails to find (below is from the build checks), but that is really quite annoying, so I propose to quiet it.
```R
33. webtide (@test_webtide.R#10) - cannot open file '/usr/local/WebTide/data/nwatl/nwatl.nod': No such file or directory

34. webtide (@test_webtide.R#10) - cannot open file '/usr/local/WebTide/data/nwatl/nwatlll.nod': No such file or directory

35. webtide (@test_webtide.R#11) - cannot open file '/usr/local/WebTide/data/nwatl/nwatl.nod': No such file or directory

36. webtide (@test_webtide.R#11) - cannot open file '/usr/local/WebTide/data/nwatl/nwatlll.nod': No such file or directory
```

**Comment by dankelley** [comment]
I don't see why this is happening. As far as I can tell, `testthat` must be reaching deep inside of R and telling it to spit out error messages, even though I am trying to open the file in a `try(..., silent=TRUE)` phrase.

**Comment by dankelley** [comment]
Fixed in `dk` commit db58b9a17107103c689656bd51efc21cc67a39d5
